src/pages/components/footer.mdx

```diff
@@ -1,15 +1,15 @@
 ---
 title: Footer
-description: The footer component page
+description: The Footer component page
 ---
 
 import { ComponentStatus } from 'components/ComponentList';
 
 <PageDescription>
 
-The footer is a section that sits at the bottom of any IBM.com page, and acts as the catch&mdash;all section that helps users navigate and provides corporate level general and legal information. The footer is required on all pages on IBM.com.
+The Footer is a section that sits at the bottom of any IBM.com page, and acts as the catch&mdash;all section that helps users navigate and provides corporate level general and legal information. The Footer is required on all pages on IBM.com.
 
-There are two different footers. All pages are required to use the default footer, with the exception of a few pages that need to minimize distraction, such as checkout process. Please [contact us](https://cognitive-app.slack.com/archives/C2PLX8GQ6) if you think your page needs approval to use the minimal footer. 
+There are two different Footers. All pages are required to use the Default Footer, with the exception of a few pages that need to minimize distraction, such as checkout process. Please [contact us](https://cognitive-app.slack.com/archives/C2PLX8GQ6) if you think your page needs approval to use the Minimal Footer. 
```

Comment:
```suggestion
There are two different Footers. All pages are required to use the default Footer, with the exception of a few pages that need to minimize distraction, such as checkout process. Please [contact us](https://cognitive-app.slack.com/archives/C2PLX8GQ6) if you think your page needs approval to use the minimal Footer. 
```

I think the `Footer` itself is capital, but the types are not capital? @wonilsuhibm ?

Reply:
I think you are right, Jeff. Wonil, can you confirm?

Reply:
Re types, yes, I agree.